runtime/themes/onelight.toml

```diff
@@ -1,129 +1,184 @@
-
-"variable" = { fg = "black" }
-"variable.builtin" = { fg = "light-blue" }
-"variable.parameter" = { fg = "red" }
-"variable.other.member" = { fg = "red" }
+"function" = { fg = "blue" }
+"function.builtin" = { fg = "cyan" }
+"function.method" = { fg = "light-blue" }
+"function.macro" = { fg = "pink", modifiers = ["blod"] }
+"function.special" = { fg = "cyan" }
 
 "markup.heading" = { fg = "red" }
-"markup.raw" = { fg = "gray" }
-"markup.raw.inline" = { fg = "green", bg = "grey-200" }
-"markup.bold" = { fg = "yellow", modifiers = ["bold"] }
-"markup.italic" = { fg = "purple", modifiers = ["italic"] }
-"markup.strikethrough" = { modifiers = ["crossed_out"] }
-"markup.list" = { fg = "light-blue" }
-"markup.quote" = { fg = "gray" }
-"markup.link.url" = { fg = "cyan", modifiers = ["underlined"] }
-"markup.link.text" = { fg = "light-blue" }
+"markup.heading.marker" = { fg = "red" }
 "markup.heading.1" = { fg = "red", modifiers = ["bold"] }
-"markup.heading.2" = { fg = "gold", modifiers = ["bold"] }
+"markup.heading.2" = { fg = "gold", modifiers = [
+  "bold",
+], underline = { style = "line" } }
 "markup.heading.3" = { fg = "yellow", modifiers = ["bold"] }
 "markup.heading.4" = { fg = "green", modifiers = ["bold"] }
 "markup.heading.5" = { fg = "blue", modifiers = ["bold"] }
 "markup.heading.6" = { fg = "purple", modifiers = ["bold"] }
+"markup.list" = { fg = "light-blue" }
+"markup.list.unnumbered" = { fg = "light-blue" }
+"markup.list.numbered" = { fg = "light-blue" }
+"markup.bold" = { fg = "yellow", modifiers = ["bold"] }
+"markup.italic" = { fg = "purple", modifiers = ["italic"] }
+"markup.link" = { fg = "light-blue" }
+"markup.link.url" = { fg = "cyan", modifiers = ["underlined"] }
+"markup.link.text" = { fg = "light-blue" }
+"markup.quote" = { fg = "grey" }
+"markup.raw" = { fg = "brown" }
+"markup.raw.inline" = { fg = "green" }
+"markup.raw.block" = { fg = "grey" }
 
-"diff.plus" = "green"
-"diff.delta" = "gold"
-"diff.minus" = "red"
-
-"diagnostic.info".underline = { color = "blue", style = "curl" } 
-"diagnostic.hint".underline = { color = "green", style = "curl" } 
-"diagnostic.warning".underline = { color = "yellow", style = "curl" } 
-"diagnostic.error".underline = { color = "red", style = "curl" } 
-
-"info" = { fg = "blue", modifiers = ["bold"] }
-"hint" = { fg = "green", modifiers = ["bold"] }
-"warning" = { fg = "yellow", modifiers = ["bold"] }
-"error" = { fg = "red", modifiers = ["bold"] }
+"diff" = { fg = "red" }
+"diff.plus" = { fg = "green" }
+"diff.minus" = { fg = "red" }
+"diff.delta" = { fg = "cyan" }
+"diff.delta.moved" = { fg = "cyan" }
 
 "ui.background" = { bg = "white" }
+"ui.background.separator" = { bg = "white" }
 
-"ui.cursor" = { fg = "white", bg = "gray" }
+"ui.cursor" = { fg = "white", bg = "grey" }
+"ui.cursor.normal" = { fg = "white", bg = "grey" }
+"ui.cursor.insert" = { fg = "white", bg = "grey" }
+"ui.cursor.select" = { fg = "white", bg = "grey" }
+"ui.cursor.match" = { bg = "light-grey" }
```

Comment:
It looks like this isn't defined in the palette. Maybe this could use one of the `grey-*` shades?

Reply:
Thanks for the review, here is a change to `light-white`, because the multi-cursor uses the grey style, here only do grayscale and multi-cursor style to distinguish.